src/schemathesis/specs/openapi/examples.py

```diff
@@ -75,9 +75,13 @@ def get_request_body_example_from_properties(endpoint_def: Dict[str, Any]) -> Di
 def get_static_parameters_from_example(endpoint: Endpoint) -> Dict[str, Any]:
     static_parameters = {}
     for name in PARAMETERS:
-        parameter = getattr(endpoint, name)
-        if parameter is not None and "example" in parameter:
-            static_parameters[name] = parameter["example"]
+        parameters = getattr(endpoint, name)
+        if name == "body" and parameters and parameters[0].example:
```

Comment:
Both branches can be unified on the "container" level. It will work if there will be a single instance, not a list